### PR TITLE
01-filedir.md: Fill in FIXME challenge titles

### PR DESCRIPTION
**Pull request by wking**
It's hard to name these in a way that's both descriptive and avoids
giving away the answer, but I've tried ;).

### CODE REVIEW

File: 01-filedir.md [hunk 1]
```diff
@@ -426,7 +426,7 @@ and we will see it in many other tools as we go on.
 > 3.  `2012-12-01/ 2013-01-08/ 2013-01-27/`
 > 4.  `original pnas_final pnas_sub`
 
-> ## FIXME {.challenge}
+> ## `ls` command construction {.challenge}
```
Comment (BernhardKonrad) [inline 01-filedir.md]: How about
Useful `ls` arguments

File: 01-filedir.md [hunk 2]
```diff
@@ -441,7 +441,7 @@ and we will see it in many other tools as we go on.
 > 3.  `ls -r -F /users/backup`
 > 4.  Either \#2 or \#3 above, but not \#1.
 
-> ## FIXME {.challenge}
+> ## `cd` arguments {.challenge}
```
Comment (BernhardKonrad) [inline 01-filedir.md]: How about
`cd` default argument

File: 01-filedir.md [hunk 3]
```diff
@@ -417,7 +417,7 @@ and we will see it in many other tools as we go on.
 
 <img src="fig/filesystem-challenge.svg" alt="Filesystem for Challenge Questions" />
 
-> ## FIXME {.challenge}
+> ## Relative path resolution {.challenge}
```
Comment (BernhardKonrad) [inline 01-filedir.md]: Great!

File: 01-filedir.md [hunk 4]
```diff
@@ -450,6 +450,7 @@ and we will see it in many other tools as we go on.
 > 3.  It changes the working directory to the user's home directory.
 > 4.  It produces an error message.
 
-> ## FIXME {.challenge}
+> ## What does `ls` do with `-s` and `-h`? {.challenge}
```
Comment (BernhardKonrad) [inline 01-filedir.md]: How about
Exploring more `ls` argument

File: 01-filedir.md (hunk 1 above)
Comment (gvwilson) [inline 01-filedir.md]: Please go ahead and send a PR with the change (and others).

Comment (BernhardKonrad) [inline 01-filedir.md]: A PR on wking's fork, or a fresh PR on swcarpentry?

Comment (r-gaia-cs) [inline 01-filedir.md]: > A PR on wking's fork, or a fresh PR on swcarpentry?

I prefer a PR on @wking's fork but I'm OK with a fresh PR on swcarpentry.

Comment (wking) [inline 01-filedir.md]: On Sat, Jan 31, 2015 at 07:38:03PM -0800, Greg Wilson wrote:
> > @@ -426,7 +426,7 @@ and we will see it in many other tools as we go on.
> >  > 3.  `2012-12-01/ 2013-01-08/ 2013-01-27/`
> >  > 4.  `original pnas_final pnas_sub`
> >  
> > -> ## FIXME {.challenge}
> > +> ## `ls` command construction {.challenge}
> 
> Please go ahead and send a PR with the change (and others).

I'm happy just re-rolling my branch ;), so I don't know that we need
another PR here.  I should get to it by tonight.

Comment (wking) [inline 01-filedir.md]: And thanks for the review :).  These PRs have been sitting here for a
while, and it's good to have things moving forward again.

Comment (BernhardKonrad) [inline 01-filedir.md]: Great, please go ahead and update your branch @wking. I'll merge when you're done.

Comment (wking) [inline 01-filedir.md]: On Sun, Feb 01, 2015 at 10:38:34AM -0800, Bernhard Konrad wrote:
> Great, please go ahead and update your branch @wking. I'll merge
> when you're done.

I've updated my branch, but I've only picked up one of your
suggestions directly.  For the other two headings where you suggested
changes, I've tried something a bit different.  Details in the commit
message for my just-pushed ba597324 (01-filedir.md: Fill in FIXME
challenge titles, 2014-12-18).

**Comment by BernhardKonrad** [comment]
Looks great to me, thanks a lot.

**Pull request merged**